microsoft/testsuites/network/networksettings.py

```diff
@@ -204,13 +206,24 @@ def validate_device_gro_lro_settings_change(self, node: Node) -> None:
 
         devices_gro_lro_settings = ethtool.get_all_device_gro_lro_settings()
 
-        for device_settings in devices_gro_lro_settings:
-            interface = device_settings.interface
-            original_gro_setting = device_settings.gro_setting
-            original_lro_setting = device_settings.lro_setting
+        for settings in devices_gro_lro_settings:
+            interface = settings.interface
+            if settings.gro_fixed and settings.lro_fixed:
+                self.__log.info(
+                    "The GRO and LRO settings are fixed and cannot be changed for"
+                    f" device {interface}. Skipping test for this device"
```

Comment:
If both of them are fixed, and not testable, it needs to raise a skipped exception.

Reply:
Yes, is it possible that all devices are fixed? Or some of them are fixed?

Reply:
Yes both scenarios are possible i.e. all devices can have settings as [fixed] and/or some of them have [fixed].

Reply:
ok. there can be a flag. If all are fixed, it should raise a skipped exception.

Reply:
A bool flag called 'can_set' is enough. The initial value is False. If any is not fixed, it will be set to True. If it's False, after the loop, raise the skipped exception.